highfidelity/hifi
# Single assignment-client process fix for: `-n 1`

This should fix `assignment-client -n 1` not to create an assignment-client monitor but just a single assignment-client class instead.

Test-Notes:
- Run a domain-server
- Spawn an assignment-client with param `-n 1` to have a single fork
- Verify that only one assignment-client process is running
